### PR TITLE
Example projects

### PR DESCRIPTION
**Pull request by icarusz**
This is just a start but it plays down tutorials a bit (since they are all JS RPC based) and creates a new Menu Item "Example Applications".
Under that there are three new subitems, Tupelo Explorer, Denentracar, Iot Lock Controller.

All those items are associated with new pages.

Screen Shot 2019-10-21 at 3 56 44 PM

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+nav_order: 3
+---
+
+# Example Projects
+
+## Tupelo Explorer and Wallet
+
+Tupelo Explorer is a WASM, react-based app built on the Tupelo WASM SDK.  
+The explorer enables users to browse ChainTrees, the data-structure underlying
+Tupelo.  Tupelo Explorer can also be used as a wallet for Chaintrees (NFTs) you
```
Comment (tobowers) [inline examples.md:14]: not really... the wallet as part of the explorer is the old one that only really does tokens.

File: examples/iot_lock.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+The react-native client can be granted temporary access with control and tracking
+all signed by the Tupelo testnet.
+
+<img style="width:515px;height:403px; padding: 15px;" src="tupelo-zwave.gif">
+
+There are wide applications for Tupelo for IoT because it is efficient,
+fast and even includes the possibility of devices being able to check transaction
+validity when they are offline.  A small example of offline functionality
+[can be found here](https://github.com/quorumcontrol/offliner) but hop into our
+developer chat or ask for more information if you have any questions.
```
Comment (tobowers) [inline examples/iot_lock.md:25]: link to chat.

**Review by tobowers — APPROVED** [review]
I'm fine with pushing and the iterating, just a few comments - the only blocker I think would be the calling the wallet in the explorer an NFT wallet.

Also - should the actual wallet be on this list?

File: examples.md (hunk 1 above)
Comment (icarusz) [reply examples.md:14]: I should have just done it then pushed it but... this change is in the next iteration branch improving-examples .

File: examples/iot_lock.md (hunk 2 above)
Comment (icarusz) [reply examples/iot_lock.md:25]: I should have just done it then pushed it but... this change is in the next iteration branch improving-examples .

**Pull request merged**